book/website/collaboration/inclusive-events/inclusive-events-schedules.md

```diff
@@ -0,0 +1,206 @@
+# Inclusive Event Schedules and Agenda
+When developing a schedule or agenda for your event, it is important to reflect on how diferent aspects of an event's schedule will impact how inclusive and accessible your event will be.
```

Comment:
Typo: When developing a schedule or agenda for your event, it is important to reflect on how **different** aspects of an event's schedule will impact how inclusive and accessible your event will be.

Reply:
Hiya Giulia! 

I have a sneaky suspicion that this sentence has been revised or changed (as I can no longer find it) 🙈